src/dev/BQ76952.cpp

```diff
@@ -227,25 +227,39 @@ BQ76952::Status BQ76952::communicationStatus() {
     return BQ76952::Status::ERROR;
 }
 
-BQ76952::Status BQ76952::getCellVoltage(uint16_t cellVoltages[NUM_CELLS], uint32_t* sum) {
+BQ76952::Status BQ76952::getCellVoltage(uint16_t cellVoltages[NUM_CELLS], uint32_t* sum, int16_t* minVoltage, uint8_t* minCellVoltageID, int16_t* maxVoltage, uint8_t* maxCellVoltageId) {
     Status status = Status::OK;
     uint8_t cellVoltageReg = CELL_VOLTAGE_BASE_REG;
 
     uint32_t currentVoltage = 0;
+    uint16_t currentMinVoltage = 4;
```

Comment:
This is where the code can be simplified a bit having a struct to store the values